### ISSUE TITLE
Create workflow to automatically prepare a Proton prefix.

### ISSUE THREAD

**Issue by shymega** [DeckCheatz/BuiltPrefixes#1]
This is a tracking issue.

We want GitHub Actions to non-interactively prepare a Proton prefix, upload to
releases (as a pre-release), and create issue for testing.

This should automate the process to around 90%, allowing maintainers to work on the project effectively.

**Comment by marvin1099** [comment]
True, gh should generate and upload, are you still working on full automation.
Maby a manual trigger to generate a proton version prefix would be nice.
If we want to use build prefixes they actually have to be build and rn they are not.

**Comment by shymega** [comment]
I am working on it still. The issue is with Proton-GE > 10-23.

I am planning to do some more work on this when I have time off.

Manual trigger already exists.

**Comment by marvin1099** [comment]
Possibly a reference point or cherry picking pr to make builds pass #4 feel free to use or not.
